src/schemas/json/sarif.json

```diff
@@ -97,6 +97,24 @@
           {
             "$ref": "#/definitions/hash"
           }
+        },
+
+        "properties": 
+        {
+          "description": "Key/value pairs that provide additional details about the file reference.",
+          "type": "object",
+          "default": {}
```

Comment:
Is there a way to specify that the default is "not present" rather than "an empty object"?